packages/coordinator/src/components/Schedule/ScheduleDay/ScheduleDayComponent.module.scss

```diff
@@ -0,0 +1,6 @@
+.scheduleDay {
+}
+
+.cellWrapper {
+  border: 3px solid var(--coordinator-background);
+}
```

Comment:
It seems to me a little bit crocked to use 3px border for gapping. maybe use a table or a grid layout?

Reply:
Great comment, changing to grid view

Reply:
Done